proto/internalapi/central/network_endpoint_with_process.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+
+option go_package = "central";
+
+import "google/protobuf/timestamp.proto";
+import "storage/network_flow.proto";
+import "storage/process_indicator.proto";
+import "internalapi/sensor/network_enums.proto";
+
+package central;
+
+message NetworkEndpointWithProcess {
```

Comment:
We do not need the listen address based on the MVP requirements. A suggested simplification would be the following:

```
message ProcessListeningOnPortsUpdate {
    repeated storage.ProcessListeningOnPort processes_listening_on_ports = 1;
    google.protobuf.Timestamp time = 2;
}

// keep this definition in storage/
message ProcessListeningOnPort {
    uint32 port = 1;
    storage.L4Protocol protocol = 2;
    storage.ProcessIndicatorUniqueKey process = 3;
    google.protobuf.Timestamp close_timestamp = 4;
}
```

Reply:
Done